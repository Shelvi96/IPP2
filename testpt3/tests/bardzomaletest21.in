NEW ID

63;790532>8:6
:6:20;82>9:911823
:>2
:724>23:
1>:789426
07>23
9:;4076;9;>46
4;12>2
241858>;4882046
;98:5082>74:5:
@628337273;137;2683813
@40;02528582505024755314
@;5086802046218::5579:;
@3690
@29316394327361:;209;54883;
@107724:4:5
@397201724931
@098840;607533
@;684190325:4:073081185
@83170827788095
@07934773;;34001509;797953
@177:;0:872825;593:94395283:90
@;54733;5367;01:5:13;:;0310:392
@97483778;1825
@1:;66:49:0::26;5604;07
