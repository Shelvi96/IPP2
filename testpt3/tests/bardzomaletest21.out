5764801
3486784401
2591956246
0
116657515782144
0
0
0
5625844704
8
0
45952870649896573
551431281075009863
1
1100100000
